handbook/strategy/objectives.md

```diff
@@ -0,0 +1,25 @@
+# Launch Pad Objectives
+
+::: warning
+This page is a work in progress.
+:::
+
+We want to create a set of clear, measurable objectives for each school year, in rough order of priority. This helps hold us accountable to our goals, ensure that we are improving the club, and learn from our experiences.
+
+Objectives and results for each year are documented in this page. Each school year is denoted by the year of the fall semester, typically in September, and ends around May of the following year. Objectives have the following format:
+
+- **Objective: some overarching theme we want to work towards**
+  - Result: some measureable result we want to achieve to make progress towards this objective
+
+## 2020
+
+- **Encourage more collaboration between developers and designers**
+  - Every single club member participates in at least 3 discussions on GitHub (issues and pull requests)
+
+- **Mentorship & diversity within the club**
+  - 40% of new members are “beginners”
+  - Every lead and 2 developers from each team participates in a cross-team discussion
```

Comment:
Just for my own clarification, what are `cross-team discussion`s?

Reply:
I've updated the wording to be a bit clearer hopefully :) 6a6274c85ab9f5a7c8fe088cae92081cc57186f8